Library/Formula/bazel.rb

```diff
@@ -21,6 +21,7 @@ def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
 
     system "./compile.sh"
+    system "./output/bazel" "build" "scripts:bash_completion"
```

Comment:
You forgot the commas between the arguments :wink: 

In this case Ruby behaves the same way as Python or C: it assumes it’s one long string. `"a" "b" == "ab"` is true in both Python and Ruby.